source/getting-started/use_cases/file_integrity.rst

```diff
@@ -98,9 +98,29 @@ Expand the output to see an example response of this use case.
 
 In the *Integrity monitoring* module of the Wazuh dashboard, users can view all of the details of the alerts triggered and find a comprehensive summary of changes in files and registry keys. 
 
+.. raw:: html
+    
+    <div class="images-rn-420-container">
+    <div class="images-rn-420">
+
+.. thumbnail:: /images/getting_started/use-cases/wazuh-use-cases-fim1.png
+      :title: Integrity monitoring dashboard
+      :align: center
+
+.. thumbnail:: /images/getting_started/use-cases/wazuh-use-cases-fim2.png
+      :title: Integrity monitoring
+      :align: center
+
+..
+   .. thumbnail:: /images/getting_started/use-cases/wazuh-use-cases-fim3.png
```

Comment:
```suggestion

```

Reply:
Lines 115 and 116 were not removed but commented to let wazuh-use-cases-fim3.png ready to be included in a future revision.